### ISSUE TITLE
EViT-DeiT-S Reproduce

### ISSUE THREAD

**Issue by Cydia2018** [youweiliang/evit#3]
Many thanks to the authors for their open source work.

I would like to ask about the specific configuration of EViT-DeiT-S for fine-tuning experiments, including --nproc_per_node, --batch-size, --warmup-epochs, --shrink_epochs, --shrink_start_epoch, etc.
![1](https://user-images.githubusercontent.com/41748724/157159353-4be085d4-fb10-419a-8a1a-ce7a60284239.PNG)

Directly following the finetune.sh configuration, I can't seem to reproduce the 78.5% accuracy.

**Comment by youweiliang** [comment]
Hi, thanks for your interests in our work. 
The reported results in the paper were obtained using a batch size of `16x128=2048`. Could you try setting `--batch-size 256` if you are using 8 GPUs?

**Comment by Cydia2018** [comment]
> Hi, thanks for your interests in our work. The reported results in the paper were obtained using a batch size of `16x128=2048`. Could you try setting `--batch-size 256` if you are using 8 GPUs?

Unfortunately, after using the batch_size of `8x256=2048`, the accuracy is still only 78.25%, do other parameters need to be adjusted?

**Comment by youweiliang** [comment]
Hi, I just figured out the bug that caused the lower accuracy, which is that the default `--shrink_start_epoch` was wrongly set to 10. I have fixed the bug by explicitly setting `--shrink_start_epoch 0` in `finetune.sh`. Thank you for pointing out the issue.
